CONTRIBUTING.md

```diff
@@ -66,7 +71,12 @@ contains the following:
   and also with Python 2 if `python2 = true` is set in `METADATA.toml` (see below).
 * (Rarely) some docs specific to a given type stub package in `README` file.
 
-When a third party stub is
+The fastest way to generate new stubs is to use [stubgen](https://mypy.readthedocs.io/en/stable/stubgen.html),
+a tool shipped with mypy. Please make sure to use the latest version.
+The generated stubs usually need some trimming of imports. You also need
+to run `black` and `isort` manually on the generated stubs (see below).
```

Comment:
It would be nice if we can make stubgen do this automatically.

Reply:
Indeed. My plan was to see whether black and/or isort are available and the run them with the appropriate options, but I never got around to implement it.